p4sdk/Makefile

```diff
@@ -2,6 +2,8 @@
 
 .PHONY: all fmt check-fmt vet build ipuplugin test update-mod ipuplugin-amd64 ipuplugin-arm64
 APP_NAME = ipu-p4-sdk
+P4_NAME = rh_mvp
```

Comment:
Can this P4_NAME be configurable as param to Makefile, default value can be rh_mvp. Like P4_NAME ?= rh_mvp